Makefile

```diff
@@ -53,7 +53,11 @@ all: build local-gadget
 phony_explicit:
 
 ebpf-objects:
-	TARGET=$(GOHOSTARCH) go generate ./...
+	docker run --rm --name ebpf-object-builder --user $(shell id -u):$(shell id -g) -v $(shell pwd):/work ghcr.io/kinvolk/inspektor-gadget-ebpf-builder
```

Comment:
break this line?

Reply:
No a big deal, I was just trying to make it more readable since IMO a 140 long line is difficult to handle.